python/py-tensorflow/Portfile

```diff
@@ -94,15 +95,6 @@ if {${name} ne ${subport}} {
             port:py${python.version}-enum34
     }
 
-    # Issue with implicit -fstack-check with Xocde 11 + Darwin 19.
-    # Current information is that this should be fixed in Xcode 11.2
-    if { ([vercmp ${os.major} 19] >= 0) && ([vercmp $xcodeversion 11.2] < 0) } {
-        if {[string match clang ${configure.compiler}]} {
-            configure.cflags-append   -fno-stack-check
-            configure.cxxflags-append -fno-stack-check
-        }
-    }
-
     use_configure yes
```

Comment:
Just for my info, Have you checked to see if this port Builds with Xcode 11.2 ?

Reply:
When I was developing this PG on Xcode 11.2 Beta, I kept running into the [separate(?) issue](https://trac.macports.org/ticket/59282), so I could never finish the build.
I am afraid I have not tried again with the official release of Xcode 11.2.

Reply:
Can I ask then you set both the tensorflow ports to have this 'fixed' in Xcode 11.3. I will lower this to 11.2 if/when I get around to trying these ports on this version. I would rather default to keeping the workaround in place, rather than turn it off, until that is done.

Reply:
> Can I ask then you set both the tensorflow ports to have this 'fixed' in Xcode 11.3. I will lower this to 11.2 if/when I get around to trying these ports on this version. I would rather default to keeping the workaround in place, rather than turn it off, until that is done.

The default fix version number is now 11.3.
So if it turns out that 11.2 is sufficient, just add `xcode_workaround.fixed_xcode_version 11.2`.